pkg/server/store/ServerDataStore.go

```diff
@@ -0,0 +1,10 @@
+package serverDataStore
+
+type ServerDataStore struct {
```

Comment:
can we write a proper function like other,  fetch store data by interface

Reply:
interface is not required here (As discussed earlier)